modules/star/1.0/star.smk

```diff
@@ -27,7 +27,7 @@ CFG = op.setup_module(
 )
 
 # Include `utils` module
-include: "../../utils/1.0/utils.smk"
+#include: "../../utils/1.0/utils.smk"
```

Comment:
I would not change version 1.0 of the `star` module. I would create a new version and update the demo accordingly. Every version is considered "frozen" once merged into master.